src/components/dashboard/FeedItems/FeedModalItem.js

```diff
@@ -33,7 +33,13 @@ const FeedModalItem = (props: FeedEventProps) => {
   const showJaggedEdge = ['claim', 'sendcompleted', 'withdraw', 'receive'].includes(itemType)
 
   return (
-    <ModalWrapper leftBorderColor={mainColor} onClose={buttonPress} showJaggedEdge={showJaggedEdge} fullHeight={true}>
+    <ModalWrapper
+      leftBorderColor={mainColor}
+      itemType={itemType}
+      onClose={buttonPress}
+      showJaggedEdge={showJaggedEdge}
+      fullHeight={true}
```

Comment:
modals should have marging at the bottom, they should not take the full height, see design.

Reply:
done

Reply:
@StanislavShevchenko the margins at the bottom are for all devices

Reply:
@sirpy I added margins at the bottom are for all devices and removed  full height